src/tools/rbd/ArgumentTypes.cc

```diff
@@ -289,7 +289,9 @@ std::string get_short_features_help(bool append_suffix) {
 
     std::string suffix;
     if (append_suffix) {
-      if ((pair.first & g_conf->rbd_default_features) != 0) {
+      uint64_t features;
```

Comment:
You are correct -- it should show the active default features not the hard-coded default features.